python/rmm/_version.py

```diff
@@ -1,566 +0,0 @@
-# This file helps to compute a version number in source trees obtained from
```

Comment:
Please also delete `.gitattributes`, which references only this file.

Reply:
Done